src/classes/ADDR_Seasonal_TEST.cls

```diff
@@ -0,0 +1,923 @@
+/*
```

Comment:
* This method is empty, I suggest to remove it.
* Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.